exporter/honeycombmarkerexporter/README.md

```diff
@@ -0,0 +1,46 @@
+@@ -0,0 +1,18 @@
+# Honeycomb Marker Exporter
+
+This exporter allows creating markers, via the Honeycomb Markers API, based on the look of incoming telemetry. 
+
+The following configuration options are supported:
+
+* `api_key` (Required): This is the API key (also called Write Key) for your Honeycomb account.
+* `api_url` (Required): You can set the hostname to send marker data to.
+* `markers` (Required): This specifies the exact configuration to create an event marker. 
+  * `markertype`: Specifies the marker type. Markers with the same type will appear in the same color in Honeycomb. MarkerType or MarkerColor should be set.  
```

Comment:
I find this a bit confusing, shouldn't it just be `type`? (Same for fields below)

Reply:
Ya the readme is a little out of date, but we'll clean it up once we start implementing the business logic and changes in the config settle down.

Reply:
Since I have to do @mx-psi other suggested change anyway, I will just refactor this to simply be `type` and `color`.

Reply:
The README is still not going to be exact, but like Tyler said I will update it in the next wave of updates.